app/Models/EntryDAO.php

```diff
@@ -2,23 +2,27 @@
 
 class FreshRSS_EntryDAO extends Minz_ModelPdo implements FreshRSS_Searchable {
 
-	public function isCompressed(): bool {
+	public static function isCompressed(): bool {
```

Comment:
I have made a few functions static so that the new `sqlBooleanSearch()` can also be static and easier to test.